src/platform/site-wide/announcements/selectors.js

```diff
@@ -11,11 +11,8 @@ export function selectAnnouncement(
   if (announcements.isInitialized) {
     announcement = config.announcements
       .filter(a => !a.disabled)
+      .filter(a => !announcements.dismissed.includes(a.name))
       .find(a => a.paths.test(path));
-
-    if (announcement && announcements.dismissed.includes(announcement.name)) {
-      announcement = null;
-    }
```

Comment:
The idea here before was that we didn't want more than one announcement rendering on the same page (one after another), so the selector should find the same announcement each time and the "dismissed" behavior should be applied at the end. However, now we have a really complex announcements config where more than one announcement renders on the same page and that page alone, it's more complicated, and I think the announcement-stacking behavior I was trying to avoid is okay now.

Reply:
Deferring to your judgement that the multiple announcement case is a non-issue